docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,408 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        Compute the top-1 discord in <code>T</code>, where the subsequence discord resides in T[split_index:]
</pre><p>We can easily improve this function to return top-k discords. However, since the implementation of DAMP in this notebook support top-1 discord for now, I decided to avoid add param <code>k</code> to the naive function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>